### DISCUSSION TITLE
Importing CSV - halts at 10%

### DISCUSSION THREAD

**Discussion by carlomullen** [snipe/snipe-it#12054] [Q&A]
Hi, 

I am a new user to Snipe-it and attempting to import a CSV, initially creating my own and using preset titles, which failed then I created another using the sample CSV supplied, and this failed as well.

I have checked server permissions and these look correct.

Not sure what else I need to check as I have taken the CSV back to just the titles and one line of content.

Just a thought, do I need to have models, and categories already populated before the CSV is imported so that they match up, if they are not already there then they won't be linked when the CSV is uploaded?

Any guidance would be much appreciated.

Thank you

**Comment by corydlamb** [comment]
Do you have all of the requisite fields in this CSV? I ask because there are some that are indeed required to do CSV imports and category is one of them. The requisite fields can be found at https://snipe-it.readme.io/docs/importing#csv-format
